src/current/_includes/v23.2/sql/transactions-limit-rows.md

```diff
@@ -0,0 +1,15 @@
+- When the `transaction_rows_read_err` [session setting]({% link {{ page.version.version }}/set-vars.md %}#transaction-rows-read-err) is enabled, transactions that read more than the specified number of rows will fail. In addition, the [optimizer]({% link {{ page.version.version }}/cost-based-optimizer.md %}) will not create query plans with scans that exceed the specified row limit. For example, to set a default value for all users at the cluster level:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER ROLE ALL SET transaction_rows_read_err = 1000;
+    ~~~
+
+- When the `transaction_rows_written_err` [session setting]({% link {{ page.version.version }}/set-vars.md %}#transaction-rows-written-err) is enabled, transactions that write more than the specified number of rows will fail. In addition, the [optimizer]({% link {{ page.version.version }}/cost-based-optimizer.md %}) will not create query plans with scans that exceed the specified row limit. For example, to set a default value for all users at the cluster level:
```

Comment:
> the optimizer will not create query plans with scans that exceed the specified row limit

Is this true for the `transaction_rows_written_err` setting as well?

Reply:
Oops; should this be amended to `the optimizer will not create query plans with writes that exceed the specified row limit`? Sorry for the oversight.

Reply:
As we discussed in Slack, the optimizer does not change plans based on the `transaction_rows_written_err` setting. So I think you should just remove the bullet point entirely.

Reply:
Thanks for confirming. I restructured a bit so that both settings are still in the `Table scan best practices` guidance, and only one is mentioned under `Prevent the optimizer from planning full scans`.

Reply:
LGTM